x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_logic.ts

```diff
@@ -198,7 +198,7 @@ export const CredentialsLogic = kea<
     shouldShowCredentialsForm: [
       false,
       {
-        showCredentialsForm: (shouldShowCredentialsForm) => !shouldShowCredentialsForm,
+        showCredentialsForm: (_) => true,
```

Comment:
super small nit: I think we can lose the `_` entirely in this case
```suggestion
        showCredentialsForm: () => true,
```